lsp-mode.el

```diff
@@ -5093,8 +5101,8 @@ It will show up only if current point has signature help."
 
 (lsp-defun lsp--execute-command ((action &as &Command :command :arguments?))
   "Parse and execute a code ACTION represented as a Command LSP type."
-  (-if-let* ((action-handler (lsp--find-action-handler command)))
-      (funcall action-handler action)
+  (-if-let* ((action-handlers (lsp--find-action-handlers command)))
```

Comment:
the IDs are unique, see https://github.com/microsoft/vscode-eslint/blob/d1a6ff2b44c0504820e44825b2e31fd6489ebbc2/client/src/extension.ts#L1484

and here are vscode docs on that:

```
		 * Registers a command that can be invoked via a keyboard shortcut,
		 * a menu item, an action, or directly.
		 *
		 * Registering a command with an existing command identifier twice
		 * will cause an error.
```

Reply:
Nice, I didn't know that